SignInCheckIn/SignInCheckIn.Tests/SignInCheckIn.Tests.csproj

```diff
@@ -117,6 +137,10 @@
       <HintPath>..\packages\NUnit.3.4.1\lib\net45\nunit.framework.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Owin, Version=1.0.0.0, Culture=neutral, PublicKeyToken=f0ebd12fd5e55cc5, processorArchitecture=MSIL">
```

Comment:
Yea - we do need to pull in SignalR and its dependencies (one being Owin) into the SignInCheckIn tests project now that we are actually unit testing websocket stuff